etc/orthography.tsv

```diff
@@ -1,472 +1,342 @@
+k	k	702	U+006B	"nɑ̄̌kɑ,puike,wi̱kitmɑl"	Tetelcingo Mexicano, Kawaiisu, Orayvi Hopi, N Tepehuan, Southern Ute, Big Smokey Valley Shoshone, Comanche, Pipil, Tubar, Western Shoshone, Rarámuri, Arizona Yoeme (Yaqui), Chemehuevi, Southern Paiute, Classical Nahuatl, Tümpisa Shoshone (Panamint,Koso), Wixarika (Huichol), Cahuilla, Pochutla Mexicano, Tongva (Gabrielino), Cora, Shoshone, Pahkannil (Tübatulabal), Mayo, Northern Paiute, Zuni, Eudeve, Western Mono, Guarijío, Kitanemuk, Cupeño, S Tepehuan, Tohono O'odham, Ópata, Pima Bajo, Luiseño, Acjachemen (Juaneño), Tewa, Ipai, Serrano
+l	l	354	U+006C	"heli,kwitɑpil,nɑ̱qɑl"	Luiseño, Tetelcingo Mexicano, Zuni, Pipil, Classical Nahuatl, Cahuilla, Rarámuri, Pahkannil (Tübatulabal), Tubar, Cupeño, Acjachemen (Juaneño), Mayo, Ipai, Arizona Yoeme (Yaqui), Guarijío, Orayvi Hopi, Pima Bajo, Pochutla Mexicano, Tohono O'odham, N Tepehuan, Cora, Wixarika (Huichol)
+m	m	502	U+006D	"tokōko moʔle,tɑmi,ɯ̄mɯhɯɁ"	Cahuilla, Tümpisa Shoshone (Panamint,Koso), Ipai, Rarámuri, Southern Paiute, Orayvi Hopi, Big Smokey Valley Shoshone, Tubar, Luiseño, Tetelcingo Mexicano, Tongva (Gabrielino), Shoshone, Kawaiisu, N Tepehuan, Guarijío, Kitanemuk, Comanche, Arizona Yoeme (Yaqui), Western Shoshone, Cupeño, Serrano, Chemehuevi, Pahkannil (Tübatulabal), Classical Nahuatl, Acjachemen (Juaneño), Pipil, Wixarika (Huichol), Western Mono, Northern Paiute, Pima Bajo, Pochutla Mexicano, Mayo, Southern Ute, Ópata, Eudeve, Zuni, Cora, Tewa, Tohono O'odham, S Tepehuan
+n	n	729	U+006E	"kulɑ̄n,nini̱r,ɑjnɑ"	Guarijío, Tetelcingo Mexicano, Kawaiisu, Big Smokey Valley Shoshone, Comanche, Western Mono, Zuni, Pahkannil (Tübatulabal), Shoshone, Pipil, Tongva (Gabrielino), Cora, Southern Ute, Tümpisa Shoshone (Panamint,Koso), Orayvi Hopi, Chemehuevi, Classical Nahuatl, Cahuilla, Cupeño, Western Shoshone, Arizona Yoeme (Yaqui), Tubar, Rarámuri, Northern Paiute, Mayo, Pochutla Mexicano, Luiseño, Southern Paiute, Wixarika (Huichol), Eudeve, Kitanemuk, N Tepehuan, Ipai, Ópata, Acjachemen (Juaneño), S Tepehuan, Serrano, Pima Bajo, Tewa, Tohono O'odham
+o	o	706	U+006F	"horo-,o̱tʃoko,topɑxɑro̱"	Tongva (Gabrielino), Tetelcingo Mexicano, Zuni, Orayvi Hopi, Kawaiisu, Big Smokey Valley Shoshone, Guarijío, Tohono O'odham, Tümpisa Shoshone (Panamint,Koso), Tubar, Classical Nahuatl, Western Shoshone, Comanche, S Tepehuan, Pima Bajo, Chemehuevi, Northern Paiute, Shoshone, Western Mono, Kitanemuk, Rarámuri, Pochutla Mexicano, Tewa, Arizona Yoeme (Yaqui), Mayo, Acjachemen (Juaneño), Southern Ute, N Tepehuan, Luiseño, Eudeve, Pahkannil (Tübatulabal), Ópata, Southern Paiute, Ser
```

Comment:
given that "kw" was a proto-ua phoneme, it is more likely that the instance `kkw`, as in your three examples, should be read as `k k_w`.

Reply:
I am not sure I follow, GitHub is showing me all your comments in terms of line 26 ("o").

But in case of labialization, the source is quite clear in this sense, as the raw data has the IPA and the latex tipa. `w` are glides and superscripts are labialized diacritics.